feat(generate): support @default jsdoc tag (#294)
bennypowers/cem: docs/content/docs/reference/commands/generate.md

These tags apply to class properties:

> - `@default` / `@defaultvalue` — Default value for the property

Reviewer: Note here that if the field has an initializer, the default is computed from there, but jsdoc wins

> - `@default` / `@defaultvalue` — Default value for the property. Use this when the property does not have a default value of itself. If a property has both, the jsdoc tag wins.

Reviewer: ```suggestion
- `@default` / `@defaultvalue` — Default value for the field. Not needed when the field has an initializer. If a field has both, the jsdoc tag wins.
```
